### PR TITLE
Input fields are align in the screening and demographic information card [INT-1582]

### PR DESCRIPTION
**Pull request by pokharelmanish**
### Jira Story

- [Date of Birth Field in Person Card are Not Aligned INT-1582](https://osi-cwds.atlassian.net/browse/INT-1582)

## Description
Input fields are aligned in the screening and demographic information card.
Please check screenshot in JIRA. 

## Tests
Not needed

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/javascript/views/ScreeningInformationForm.jsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ const ScreeningInformationForm = ({
         onChange={(value) => onChange('ended_at', value)}
       />
       <SelectField
-        gridClassName='col-md-4'
+        gridClassName='col-md-4 align-communciation-method'
```
Comment (brunoOSI) [inline app/javascript/views/ScreeningInformationForm.jsx]: You have a typo, should be communication.

File: app/javascript/views/people/PersonDemographicsForm.jsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ const PersonDemographicsForm = ({
     </div>
     <div className='row'>
       <DateField
-        gridClassName='col-md-3'
+        gridClassName='col-md-3 align-dob'
```
Comment (brunoOSI) [inline app/javascript/views/people/PersonDemographicsForm.jsx]: One recommendation would be to look for situations where this CSS class can be useful besides this specific field.
Maybe you can spot other date fields that suffer the same issue as part of this story and create an `date-field-alignment-correction` or something less specific.

Comment (NagaPradeepGajula-osi) [reply app/javascript/views/people/PersonDemographicsForm.jsx]: Same recommendation as Bruno mentioned.

Comment (pokharelmanish) [reply app/javascript/views/people/PersonDemographicsForm.jsx]: :+1:

**Comment by pokharelmanish** [comment]
retest this please

**Pull request merged**